lib/tasks/data_fixes.rake

```diff
@@ -1,4 +1,4 @@
-require 'tasks/data_clean_up/move_user_recorded_data_to_registration_facility'
+require 'tasks/scripts/move_user_recorded_data_to_registration_facility'
```

Comment:
Does this need to be in this PR?

Reply:
Don't want to have multiple directories with just one file in them.